specification/resourcemover/resource-manager/Microsoft.Migrate/stable/2021-01-01/resourcemovercollection.json

```diff
@@ -1285,10 +1366,9 @@
           "description": "Gets or sets the resource settings.",
           "$ref": "#/definitions/ResourceSettings"
         },
-        "sourceResourceSettings": {
-          "description": "Gets or sets the source resource settings.",
-          "$ref": "#/definitions/ResourceSettings",
-          "readOnly": true
+        "targetResourceName": {
```

Comment:
this is optional here, will RMO assume the source resource and target resource name to be same if not provided?